### DISCUSSION TITLE
Update meeting minutes after import

### DISCUSSION THREAD

**Discussion by dku** [ietf-tools/datatracker#6175] [Help / Questions] [answered]
Hello,

I might be missing something, but how can update meeting minutes after I imported them from HedgeDoc?

Thanks,
Dirk

**Comment by rjsparks** [comment, marked as answer]
You can edit them on the meeting materials page.
On https://datatracker.ietf.org/meeting/117/session/stir, for example:
![image](https://github.com/ietf-tools/datatracker/assets/10996692/8a9c6474-7c87-496a-aad8-1bca6f24a773)
Note the `Upload revision` and `Import from notes.ietf.org` options.

You can download the minutes, edit them, and then use the `Upload revision` option.
Or you can edit the notes.ietf.org page, and re-import using the `Import from notes.ietf.org` option.

**Comment by dku** [comment]
ACK – thanks!

Dirk
